motivations.rst

```diff
@@ -193,7 +213,7 @@ participating in the CPython core development process:
 
 .. topic:: Barry Warsaw (United States)
 
-   * `LinkedIn <https://www.linkedin.com/in/barry-warsaw/>`_ (Staff Software Engineer - Python Foundation)
+   * LinkedIn: `<https://www.linkedin.com/in/barry-warsaw/>`_ (Staff Software Engineer - Python Foundation)
```

Comment:
Note that reST doesn't allow multiple links with the same title :-(

Reply:
Didn't realize that before.

Reply:
It's due to the reverse anchors docutils normally creates - you can turn those off for a link by using a double trailing underscore instead of a single trailing underscore.